cfme/provisioning.py

```diff
@@ -188,6 +189,9 @@ def select_security_group(sg):
             ('hardware_monitoring', {
                 version.LOWEST: ui.Select('select#hardware__monitoring'),
                 '5.5': AngularSelect('hardware__monitoring')}),
+            ('boot_disk_size', {
+                version.LOWEST: ui.Select('select#hardware__boot_disk_size'),
+                '5.5': AngularSelect('hardware__boot_disk_size')}),
```

Comment:
Yep. We don't care about `5.5` anymore. Let's just use the `AngularSelect` directly. Feel free to update the other `5.5` version picks as well.